homework/01LessonOne.js

```diff
@@ -145,55 +144,56 @@ var notNotUndefined;
 //
 // Your Answer Goes Here:
 //
-// ****
-
+// **** truthiness is when a value has been assigned and falsiness is when a value has not been assigned.
```

Comment:
Let's talk a bit more about this! Truthy/falsy isn't quite the same as assigned/undefined